packages/core-components/src/components/EmptyState/MissingAnnotationEmptyState.tsx

```diff
@@ -53,16 +53,21 @@ const useStyles = makeStyles<BackstageTheme>(
   { name: 'BackstageMissingAnnotationEmptyState' },
 );
 
-export function MissingAnnotationEmptyState(props: Props) {
-  const { annotation, readMoreUrl } = props;
+export function MissingAnnotationEmptyState({
+  annotation,
+  readMoreUrl,
+}: Props) {
```

Comment:
Do not unpack props directly in exported components. The API report becomes messy.

You can revert this line and instead do `const { annotation, readMoreUrl } = props;` inside the component